perfkitbenchmarker/providers/gcp/gce_network.py

```diff
@@ -226,6 +229,13 @@ def __init__(self, network_spec):
     super(GceNetwork, self).__init__(network_spec)
     self.project = network_spec.project
     name = FLAGS.gce_network_name or 'pkb-network-%s' % FLAGS.run_uri
+
+    # add support for zone, cidr, and separate networks
+    if network_spec.zone and network_spec.cidr:
+      name = FLAGS.gce_network_name or 'pkb-vpnnetwork-%s-%s' % (network_spec.zone, FLAGS.run_uri)
```

Comment:
this is not necessarily a vpn network?
maybe stick with pkb-network-...

Reply:
fixed.